test/unit/algorithm_mode/test_custom_metrics.py

```diff
@@ -12,15 +12,16 @@
 # language governing permissions and limitations under the License.
 import numpy as np
 import xgboost as xgb
-
+from math import log
 from sagemaker_xgboost_container.metrics.custom_metrics import accuracy, f1, mse
```

Comment:
Nit: `mse` should probably be removed if unit test on L50 is removed?

Reply:
You should add a mse test on regression prediction.